src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,93 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        testDriver();
+        testManufacturer();
+    }
+
+    private static void testManufacturer() {
+        Manufacturer firstManufacturer = new Manufacturer();
+        firstManufacturer.setName("Dodge");
+        firstManufacturer.setCountry("USA");
+        Manufacturer secondManufacturer = new Manufacturer();
+        secondManufacturer.setName("Tavria");
+        secondManufacturer.setCountry("Ukraine");
+        Manufacturer thirdManufacturer = new Manufacturer();
+        thirdManufacturer.setName("Toyota");
+        thirdManufacturer.setCountry("Japan");
+
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+        //create
+        System.out.println(manufacturerService.create(firstManufacturer));
+        System.out.println(manufacturerService.create(secondManufacturer));
+        System.out.println(manufacturerService.create(thirdManufacturer));
+
+        //get
+        System.out.println("--------------------------------------");
+        System.out.println(manufacturerService.get(10L));
```

Comment:
let's use `manufacturer.getId()` instead of `10L`
for example, if I run this code on my laptop it will not work. I don't have manufacturer with such id.

fix these everywhere